runtime-modules/membership/src/lib.rs

```diff
@@ -544,6 +565,23 @@ decl_module! {
                 membership.invites = membership.invites.saturating_sub(1);
             });
 
+            // Decrease the working group balance.
+            let new_wg_budget = current_wg_budget - default_invitation_balance;
```

Comment:
This is perfectly safe? I have not looked at the safemath stuff, so just alerting you to the arithmetic going on.

Reply:
Changed to saturating_sub.